### PR TITLE
[INLONG-10008][Dashboard][Manager] Supplement fields of MongoDB data source for Dashboard

### LINKED ISSUE #10008: [Improve][Dashboard][Manager] Supplement fields of MongoDB data source for Dashboard
### Description

Supplement fields of MongoDB data source for Dashboard.

### InLong Component

InLong Dashboard

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haifxu**
### Prepare a Pull Request

- Fixes #10008 

### Motivation

Supplement fields of MongoDB data source for Dashboard

### Modifications

1. Add fields of MongoDB data source for Dashboard
2. Add fields of MongoDB data source for the Manager

### Verifying this change

image

### CODE REVIEW

**Review by dockerzhang — DISMISSED** [review]

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/mongodb/MongoDBSource.java [hunk 1]
```diff
@@ -58,6 +58,9 @@ public class MongoDBSource extends StreamSource {
     @ApiModelProperty("MongoDB collection name")
     private String collection;
 
+    @ApiModelProperty("Debezium snapshot mode")
```
Comment (baomingyu) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/mongodb/MongoDBSource.java:61]: comment suggest like this “Debezium snapshot mode. Available options are initial and never, default is initial” in MongoDBSourceRequest.java

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/mongodb/MongoDBSourceDTO.java [hunk 2]
```diff
@@ -57,6 +57,9 @@ public class MongoDBSourceDTO {
     @ApiModelProperty("MongoDB collection name")
     private String collection;
 
+    @ApiModelProperty("Debezium snapshot mode")
```
Comment (baomingyu) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/mongodb/MongoDBSourceDTO.java:60]: ditto

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/mongodb/MongoDBSource.java (hunk 1 above)
Comment (haifxu) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/mongodb/MongoDBSource.java]: Done.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/mongodb/MongoDBSourceDTO.java (hunk 2 above)
Comment (haifxu) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/mongodb/MongoDBSourceDTO.java]: Done

**Review by dockerzhang — APPROVED** [review]

**Review by leezng — APPROVED** [review]

**Pull request merged**